lib/services/v2.0/nomination_pools/interfaces.dart

```diff
@@ -473,7 +465,6 @@ mixin KC2NominationPoolsInterface on ChainApiProvider {
   }
 
   /// Returns list of nomination pools.
-  /// @holyGrease - are only open pools returned or all of them?
```

Comment:
This method returns all pools in the `Pool` class you can find field `PoolStatte state` use it to determine if pool open or not